### PR TITLE
New Test Class added to ty1

### PR DESCRIPTION
**Pull request by basavaraj8767**
please review and merge it

### CODE REVIEW

File: Fireflink/src/test/java/ty1/GitTest.java
Comment (Basavaraj-A-Patil) [inline Fireflink/src/test/java/ty1/GitTest.java:1]: Add Verification steps

**Review by Basavaraj-A-Patil — COMMENTED** [review]
Fix the review comments

File: Fireflink/src/test/java/ty1/GitTest.java
Comment (basavaraj8767) [reply Fireflink/src/test/java/ty1/GitTest.java:1]: Fixed

**Comment by basavaraj8767** [comment]
Merged

**Comment by Basavaraj-A-Patil** [comment]
Merged

**Pull request merged**